api/rosetta/data_integration_test.go

```diff
@@ -0,0 +1,314 @@
+// Copyright 2021 Alvalor S.A.
```

Comment:
Careful, don't forget to update the copyright of all new files after rebasing on master

Reply:
Nice catch, I've fixed that in the other PR (this will receive the change after rebasing) :D